src/MFRC522v2.cpp

```diff
@@ -1,6 +1,9 @@
 /* SPDX-License-Identifier: LGPL-2.1 */
 #include "MFRC522v2.h"
 
+// On 32-bit processors, software CRC computation is more efficient than
```

Comment:
Can you provide some numbers? For me it seems just additional code to maintain and source of error.